tests/Integration/Behaviour/Features/Scenario/Product/SpecificPrice/delete_specific_price.feature

```diff
@@ -0,0 +1,39 @@
+# ./vendor/bin/behat -c tests/Integration/Behaviour/behat.yml -s product --tags add-specific-price
+@restore-products-before-feature
+@restore-currencies-after-feature
+@delete-specific-price
+@specific-prices
+Feature: Delete a product specific price from Back Office (BO)
+  As a BO user
+  I need to be able to delete a product specific price from BO
+
+  Background:
```

Comment:
Since you only have one scenario I'm not sure you need a Background here, you can put everything in the single scenario